assets/js/langpack/es.json

```diff
@@ -0,0 +1,126 @@
+{
+  "token": {
+    "Claim control of your":
+    "Obtén el control de tus",
+
+    "Docker images":
+    "Imágenes Docker",
+
+    "View on GitHub":
+    "Ver en GitHub",
+
+    "Download .zip":
+    "Descargar .zip",
+
+    "Download .tar.gz":
+    "Descargar .tar.gz",
+
+    "What is Portus":
+    "Qué es Portus",
+
+    "Portus is an open source authorization service and user interface for the next generation Docker Registry.":
+    "Portus es un servicio de autorización de libre código e interfaz de usuario para la próxima generación de Registros Docker",
+
+    "Why use Portus":
+    "Por qué Portus",
+
+    "The Docker Hub is a wonderful place, but what if you cannot, or do not wish to push your images to a thrid party registry? You may setup a private instance of the Docker registry, but then everyone in your organization has push and pull privileges over all of your images.":
```

Comment:
Misspelling of "third" that I just noticed  - Apparently I forgot to spell check when I did #211